src/target/riscv/riscv-013.c

```diff
@@ -67,6 +66,12 @@ static int read_memory(struct target *target, target_addr_t address,
 		uint32_t size, uint32_t count, uint8_t *buffer, uint32_t increment);
 static int write_memory(struct target *target, target_addr_t address,
 		uint32_t size, uint32_t count, const uint8_t *buffer);
+typedef enum {
+	HALTGROUP,
+	RESUMEGROUP
+} grouptype_t;
```

Comment:
Style detail - there can be blank lines around this typedef:

```suggestion

typedef enum {
	HALTGROUP,
	RESUMEGROUP
} grouptype_t;

```

Reply:
I agree with empty line before, but I like not having it after because it closely puts the enum with the function, which is the only reason the enum exists.